tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType7_ES5.ts

```diff
@@ -6,9 +6,9 @@ interface I<T> {
 declare function foo<T>(obj: I<T>): T
 
 foo({
-    p: "",

```

Comment:
Unfortunately `I<T>` has a numeric indexer so `p` causes an excess property error -- it's not numeric. The computed properties don't just because we don't check them.

Basically, we need to know the purpose of this test. Is it to check what is assignable to a type with a numeric-indexer type? If not, probably `I<T>` should have a string indexer instead. @RyanCavanaugh can you take a look?